softmmu/vl.c

```diff
@@ -1756,6 +1756,8 @@ static void version(void)
 #endif
 #elif defined(TARGET_RISCV64)
     printf("Compiled for RISCV64 (with CHERI)\n");
+#elif defined(TARGET_AARCH64)
+    printf("Compiled for AARCH64 (with CHERI)\n");
 #else
 #error "INVALID CHERI target"
 #endif
```

Comment:
The line below says:
```C
    printf("Built with C0 as NULL register\n");
```
which should be  hoisted into the TARGET_MIPS define (it's also _true_ for RISC-V but a bit meaningless), or just deleted given we've relied on this for years now.

Reply:
Its gone.